zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/UserTaskController.java

```diff
@@ -121,4 +127,36 @@ private CompletableFuture<ResponseEntity<Object>> updateUserTask(
                 .withAuthentication(RequestMapper.getAuthentication())
                 .updateUserTask(request.userTaskKey(), request.changeset(), request.action()));
   }
+
+  @PostMapping(
+      path = "/user-tasks/search",
+      produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_PROBLEM_JSON_VALUE},
+      consumes = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<Object> searchUserTasks(
```

Comment:
❌ We should add test cases to the `UserTaskControllerTest` to ensure the mapping of REST search requests to the query API works as expected. It should at least cover all the cases we have explicit (in code) or implicit (through annotations or the data model) validation for in the REST API.

Reply:
I'm working in this in a separate branch @tmetzke